src/client/broadcastClient.ts

```diff
@@ -1,44 +1,42 @@
 import { Client, ClientOptions } from ".";
 
-class BroadcastClient extends Client {
-  ledgerVersion: number | undefined = undefined;
-  private readonly _clients: Client[];
+/**
+ *
+ */
+export default class BroadcastClient extends Client {
+  private readonly clients: Client[];
 
-  constructor(servers, options: ClientOptions = {}) {
+  /**
+   * Creates a new BroadcastClient.
+   *
+   * @param servers - An array of names of servers.
+   * @param options - Options for the clients.
+   */
+  public constructor(servers: string[], options: ClientOptions = {}) {
     super(servers[0], options);
 
     const clients: Client[] = servers.map(
       (server) => new Client(server, options)
     );
 
     // exposed for testing
-    this._clients = clients;
-    this.getMethodNames().forEach((name) => {
-      this[name] = function () {
-        // eslint-disable-line no-loop-func
-        return Promise.race(
-          clients.map((client) => client[name](...arguments))
-        );
-      };
+    this.clients = clients;
+    this.getMethodNames().forEach((name: string) => {
+      this[name] = async (...args): Promise<unknown> =>
+        // eslint-disable-next-line max-len -- Need a long comment, TODO: figure out how to avoid this weirdness
+        /* eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call -- Types are outlined in Client class */
+        Promise.race(clients.map(async (client) => client[name](...args)));
     });
 
     // connection methods must be overridden to apply to all client instances
-    this.connect = async function () {
-      await Promise.all(clients.map((client) => client.connect()));
+    this.connect = async (): Promise<void> => {
+      await Promise.all(clients.map(async (client) => client.connect()));
     };
-    this.disconnect = async function () {
-      await Promise.all(clients.map((client) => client.disconnect()));
+    this.disconnect = async (): Promise<void> => {
+      await Promise.all(clients.map(async (client) => client.disconnect()));
     };
-    this.isConnected = function () {
-      return clients.map((client) => client.isConnected()).every(Boolean);
-    };
-
-    // synchronous methods are all passed directly to the first client instance
-    const defaultClient = clients[0];
-    const syncMethods = ["sign"];
-    syncMethods.forEach((name) => {
-      this[name] = defaultClient[name].bind(defaultClient);
-    });
```

Comment:
Why do we no longer need to pass in these methods to the first client?

Reply:
We're removing `sign` from the client anyways, with the `Signer`.